translate/src/modules/machinery/components/source/GoogleTranslation.tsx

```diff
@@ -45,42 +60,54 @@ export function GoogleTranslation({
     setDropdownOpen(false);
   };
 
-  // TODO: Localize selectedOptionText before setting it as selected option.
-  let selectedOptionText: string;
+  let selectedOptionElement;
   switch (selectedOption) {
     case 'alternative':
-      selectedOptionText = 'REPHRASED';
+      selectedOptionElement = (
+        <Localized id='machinery-GoogleTranslation--option-rephrase-selected'>
+          <span className='selected-option'>REPHRASED</span>
+        </Localized>
+      );
```

Comment:
This switch is unnecessarily verbose. Just set an `l10nId` value in each case, and leave out the fallback that'll be filled out by Fluent in any case. Then the element itself can be defined normally in the return value.